scalyr_agent/config_main.py

```diff
@@ -1590,20 +1570,19 @@ def configure_agent_service_for_systemd_management():
 
     # Add Linux specific options
     if sys.platform.startswith("linux"):
-        parser.add_option(
-            "",
+        parser.add_argument(
             "--systemd-managed",
             dest="systemd_managed",
             action="store_true",
             default=False,
             help="Configure the agent to be managed by systemd instead of init.d which is a default",
         )
 
-    (options, args) = parser.parse_args()
-    if len(args) > 1:
-        print("Could not parse commandline arguments.", file=sys.stderr)
-        parser.print_help(sys.stderr)
-        sys.exit(1)
+    options = parser.parse_args(args=argv)
+    # if len(args) > 1:
```

Comment:
Is this a testing change, or?

Reply:
Oops, sorry. It is a leftover from the previous command line handling logic, where there was `optparser` instead of `argparse`